setup.py

```diff
@@ -123,7 +123,7 @@
         # ** Dependencies maintained by ApeWorX **
         "eip712>=0.2.1,<0.3",
         "ethpm-types>=0.5.3,<0.6",
-        "evm-trace==0.1.0a21",
+        "evm-trace>=0.1.0a22",
```

Comment:
We normally don't leave alpha pins wide, but also this should transition into beta at this point

Reply:
What do you suggest the version constraint be?

Reply:
For an alpha it should be `==` but as we discussed in discord, this is fine for now (and we should make a full release of this package so we can do proper pins on minor revisions)

Reply:
this was here for a reason!!! lol

Reply:
oh heck ya, web3.py has been updated! awesome, thank you

Reply:
we had this pin because we had released evm-trace a bit too early where it causes install hangs on ape, but i think with the latest ETH based releases are good